internal/app/orchestrator/orchestrator_test.go

```diff
@@ -141,6 +151,9 @@ func TestGoldenPath(t *testing.T) {
 	}
 
 	respChannel, cancelWatch := orchestrator.CreateWatch(req)
+	snap := mockScope.Snapshot()
+	counters := snap.Counters()
+	testutils.AssertCounterValue(t, counters, "mock_orchestrator.downstream.create_channel", 1)
```

Comment:
out of curiosity do we have to worry about flushing stats when testing these?

Reply:
not for tests since we're using a mock scope. Basically the calls to get a snapshot "do" the job of flushing.

Reply:
awesome!